### ISSUE TITLE
weight load error

### ISSUE THREAD

**Issue by JankYang12** [holyseven/PSPNet-TF-Reproduce#27]
Finetuning Process: not import resnet_v1_50/block3/unit_7/weights:0 >
< Finetuning Process: not import resnet_v1_50/block3/unit_7/BatchNorm/beta:0 >
< Finetuning Process: not import resnet_v1_50/block3/unit_7/BatchNorm/gamma:0 >
< Finetuning Process: not import resnet_v1_50/aux_logits/weights:0 >
< Finetuning Process: not import resnet_v1_50/aux_logits/biases:0 >
< Finetuning Process: not import resnet_v1_50/psp/pool6/weights:0 >
< Finetuning Process: not import resnet_v1_50/psp/pool6/BatchNorm/beta:0 >
< Finetuning Process: not import resnet_v1_50/psp/pool6/BatchNorm/gamma:0 >
< Finetuning Process: not import resnet_v1_50/psp/pool3/weights:0 >
< Finetuning Process: not import resnet_v1_50/psp/pool3/BatchNorm/beta:0 >
< Finetuning Process: not import resnet_v1_50/psp/pool3/BatchNorm/gamma:0 >
< Finetuning Process: not import resnet_v1_50/psp/pool2/weights:0 >
< Finetuning Process: not import resnet_v1_50/psp/pool2/BatchNorm/beta:0 >
< Finetuning Process: not import resnet_v1_50/psp/pool2/BatchNorm/gamma:0 >
< Finetuning Process: not import resnet_v1_50/psp/pool1/weights:0 >
< Finetuning Process: not import resnet_v1_50/psp/pool1/BatchNorm/beta:0 >
< Finetuning Process: not import resnet_v1_50/psp/pool1/BatchNorm/gamma:0 >
< Finetuning Process: not import resnet_v1_50/block4/unit_4/weights:0 >
< Finetuning Process: not import resnet_v1_50/block4/unit_4/BatchNorm/beta:0 >
< Finetuning Process: not import resnet_v1_50/block4/unit_4/BatchNorm/gamma:0 >
< Finetuning Process: not import resnet_v1_50/logits/weights:0 >
< Finetuning Process: not import resnet_v1_50/logits/biases:0 >
< Succesfully loaded fine-tune model from ./z_pretrained_weights/resnet_v1_50.ckpt. >

< training process begins >

loss or weight norm is nan. Training Stopped!

**Comment by JankYang12** [comment]
Oh, I solved it. just because my batch_size is too small
